index.html

```diff
@@ -30,7 +30,7 @@
                     <a href="#">Contact</a>
                 </li>
                 <li class="toggle">
-                    <a href="#"><i class="fa-solid fa-bars"></i></a>
+                    <a href="#" aria-label="Navigation menu icon"><i class="fa-solid fa-bars"></i></a>
                 </li>
             </ul>
         </nav>
```

Comment:
- [ ] Please consider adding `alt` attribute to every image in the HTML code with appropriate alternative text, that's useful for interacting or understanding the page has an alt text

Reply:
Hello @Ismailco, 
Please could you explain this better, we are confused because we've already added the aria-label, which is similar to the alt attribute, but for elements like buttons, hotspots, and navigation.
Thanks.

Reply:
@VaneCode, you need to add `alt` it's not the same as `aria-label`,
eg. `<img src="./image.png" alt="logo">`

Reply:
@Ismailco Thank u for pointing this out! But this is an `a` tag that we're using so it's not possible to add an 'alt' attribute, could you point us out the `img `tag you commented about?